data/data-pipeline/data_pipeline/comparison_tool/output/nyc_dacs/nyc_dacs__tract_comparison__2022-03-15.ipynb

```diff
@@ -0,0 +1,1779 @@
+     "exception": false,
+     "start_time": "2022-03-16T00:18:19.323435",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# papermill_description=Population_stats\n",
+    "population_weighted_stats = pd.concat(\n",
+    "    [\n",
+    "        utils.construct_weighted_statistics(\n",
+    "            joined_frame,\n",
+    "            COMPARATOR_COLUMN,\n",
+    "            DEMOGRAPHIC_COLUMNS + ADDITIONAL_DEMO_COLUMNS,\n",
+    "            TOTAL_POPULATION_COLUMN,\n",
+    "        ),\n",
+    "        utils.construct_weighted_statistics(\n",
+    "            joined_frame,\n",
+    "            SCORE_COLUMN,\n",
+    "            DEMOGRAPHIC_COLUMNS + ADDITIONAL_DEMO_COLUMNS,\n",
+    "            TOTAL_POPULATION_COLUMN,\n",
+    "        ),\n",
+    "    ],\n",
+    "    axis=1,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a9fb4dfa-876d-4f25-9552-9831f51ec05c",
+   "metadata": {
+    "papermill": {
+     "duration": 0.017355,
+     "end_time": "2022-03-16T00:18:19.942993",
+     "exception": false,
+     "start_time": "2022-03-16T00:18:19.925638",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "source": [
+    "## Print to excel"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "67e2c71e-68af-44b4-bab0-33bc54afe7a4",
+   "metadata": {
+    "papermill": {
+     "duration": 0.070755,
+     "end_time": "2022-03-16T00:18:20.029526",
+     "exception": false,
+     "start_time": "2022-03-16T00:18:19.958771",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# papermill_description=Writing_excel\n",
+    "utils.write_single_comparison_excel(\n",
+    "    OUTPUT_EXCEL,\n",
+    "    population,\n",
+    "    tract_level_by_identification,\n",
+    "    population_weighted_stats,\n",
+    "    tract_level_by_grouping_formatted,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b89484f8-e705-4eb1-8983-55f899b6d218",
+   "metadata": {},
+   "source": [
+    "## Additional analysis (by hand)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "id": "ae278892-79bf-4e29-a5ad-78eba98b73a8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ny_df = joined_frame[joined_frame[\"GEOID10_TRACT\"].str.startswith(\"36\")]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "id": "3f31d3c3-4905-443e-bc25-e78e29646105",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def _get_ny_only(df, col_list):\n",
+    "    return pd.concat(\n",
+    "        [\n",
+    "            df.groupby([SCORE_COLUMN])[col_list]\n",
+    "            .mean()\n",
+    "            .T.rename(columns={False: \"Not CEJST\", True: \"CEJST\"}),\n",
+    "            df.groupby([COMPARATOR_COLUMN])[col_list]\n",
+    "            .mean()\n",
+    "            .T.rename(columns={False: \"Not NYC\", True: \"NYC\"}),\n",
```

Comment:
Nit: it's not "NYC", right? It's more "NYS" or "NY DACs"?

Reply:
yes!

Reply:
good catch. there's nyc and ros (rest of state)